### ISSUE TITLE
Feature request: Safari extension "Open link in IINA" option

### ISSUE THREAD

**Issue by bdougherty** [iina/iina#3353]
**What you want IINA to do:** Have an option in the Safari right-click menu to open a link in IINA

**What IINA does currently:** There is an option to open the current page in IINA

**Why you think this should be added:** Something my ad blocker is doing prevents most YouTube videos from working, so I use IINA to watch most videos. It would be much more convenient to be able to directly open a video url without having to load the video page first (from a channel page, for example), or without having to copy the url and then switch to IINA and open the url there.

**Comment by saagarjha** [comment]
Is this something that goes beyond the existing app extension for Safari that IINA ships with?

**Comment by bdougherty** [comment]
Yes, the current one that is included in 1.1.2 Build 128 (Safari 14.0.3) only has the option to open the current page in IINA. The request is to add an option when right-clicking on a link to be able to open that link's url in IINA.
